tests/suites/os/suite.js

```diff
@@ -298,17 +298,19 @@ module.exports = {
 			this.link,
 		);
 
-		this.log('Waiting for device to be reachable');
-		await this.utils.waitUntil(async () => {
-			this.log("Trying to ssh into device");
-			let hostname = await this.context
-			.get()
-			.worker.executeCommandInHostOS(
-			  "cat /etc/hostname",
-			  this.link
-			)
-			return (hostname === this.link.split('.')[0])
-		}, true);
+		await test.resolves(
+			this.utils.waitUntil(async () => {
+				test.comment(`Trying to SSH into device ${this.link}`);
```

Comment:
I've been trying to avoid the use of `test.comment` as it logs asynchronously and can be very confusing when looking at timestamps or out-of-order messages.

I find the `Device should be...` pass/fail to be more than enough. If we are struggling to figure out what test is currently running, we can look at the test source.